OpenWeather/Data/Network/LocationData.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  OpenWeather template generated by OpenBytes on 22/03/2023.
+//
+// Created by Ahmed Shendy.
+//  LocationData.swift
+//
+
+import Foundation
+import CoreLocation
+import CryptoKit
+
+struct LocationData {
+    let name: String
+    let location: CLLocation
+
+    var lat: Double { location.coordinate.latitude }
+    var long: Double { location.coordinate.longitude }
+
+    init(name: String, location: CLLocation) {
+        self.name = name
+        self.location = location
+    }
+}
+
+extension LocationData: CustomStringConvertible {
+    var description: String {
+        """
+{
+    name: \(name),
+    latitude: \(lat),
+    longitude: \(long)
+}
+"""
```

Comment:
I think we can align the triple quotation marks and it won't add any tabs or spacing.